server/routes/api/v1/adoptablePetsRouter.js

```diff
@@ -14,4 +14,24 @@ adoptablePetsRouter.get("/", async (req, res) => {
   }
 })
 
+adoptablePetsRouter.get("/:type/:id", async (req, res) => {
+  try {
+    const adoptablePet = await AdoptablePet.findById(req.params.id);
+    if (adoptablePet) {
+      console.log(req.params)
+      adoptablePet.petType = await adoptablePet.petType();
+      if (req.params.type === adoptablePet.petType.type) {
```

Comment:
If we're trying to view one adoptable pet, why do we have to check it's type? I think this can be removed, along with removing `:type` from the endpoint path. If we're querying for a pet, it should already be in our database and have an associated pet type, so we shouldn't have to check the type before viewing. There's also a fallback 404 here if the findById fails, so this extra check seems a little unnecessary